src/components/SideBar/menuItems.js

```diff
@@ -0,0 +1,176 @@
+import { StyledSvg } from "./style";
+
+export const menuItemsArr = [
+  {
+    path: "/dashboard/Overview",
+    name: "Overview",
+    icon: (
+      <StyledSvg
```

Comment:
Same regarding svgs and also better to add styles to wrapping component under like
```
SomeWrappingComponent = styled.div`
  svg {
    ...
  }
`
```